test/unit/spec/v2/view-builder/views/IdentifierView_spec.js

```diff
@@ -49,4 +54,29 @@ describe('v2/view-builder/views/IdentifierView', function() {
     expect(testContext.view.$el.find('.siw-main-footer .js-forgot-password').length).toEqual(1);
     expect(testContext.view.$el.find('.links-primary .js-forgot-password').length).toEqual(0);
   });
+
+  it('view renders IDP buttons correctly with idpDisplay property', function() {
+    jest.spyOn(AppState.prototype, 'hasRemediationObject').mockReturnValue(true);
+    jest.spyOn(AppState.prototype, 'getActionByPath').mockReturnValue(true);
+    jest.spyOn(AppState.prototype, 'isIdentifierOnlyView').mockReturnValue(false);
+    testContext.init();
+
+    // The idp buttons should be rendered below the main login fields when no idpDisplay defined.
```

Comment:
also add a check that if there are no buttons configured then do not render any buttons..